Assets/Scripts/EOSLobbyManager.cs

```diff
@@ -32,6 +32,8 @@
 
 namespace PlayEveryWare.EpicOnlineServices.Samples
 {
+    public enum LobbyChangedEvent { Create, Join, Leave, Kicked }
```

Comment:
My suggestion would be to utilize the following delegate and event, updating the corresponding places within this PR that would need to reference it:

(these would be inside `EOSLobbyManager`, and comments have been excluded for the sake of brevity):

```csharp

public enum LobbyChangeType
{
    Create,
    Join,
    Leave,
    Kicked
}

public delegate void LobbyChangedEventHandler(string lobbyId, LobbyChangeType typeOfChange);

public event LobbyChangedEventHandler OnLobbyChanged;

Reply:
Thank you for the expressive example! I'll implement it basically as you have here.

I'm impressed by your use of the markdown for comments, I need to start utilizing this.